### ISSUE TITLE
Failing to parse date on Ubuntu 14.04.2 LTS

### ISSUE THREAD

**Issue by keithpitt** [spotify/docker-gc#30]
When I run `docker-gc` on Ubuntu 14.04.2 LTS, I get the following error message:

```
date: invalid date ‘"2015-07-29 16:16:2’
/usr/sbin/docker-gc: line 69: 1438186643 - : syntax error: operand expected (error token is "- ")
```

Am I doing something wrong? Thanks!

**Comment by mbruggmann** [comment]
This is likely caused by the same issue as https://github.com/spotify/docker-gc/issues/29.

**Comment by rohansingh** [comment]
Should be fixed like #29.
